<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutSetDialog</class>
 <widget class="QDialog" name="ShortcutSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditScreenShot">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditStickyNote">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditColorPicker">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTaskManager">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMainTool">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditReminder">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHotkeyManager">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditStock">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHardware">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelScreenShot">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>截图工具：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStickyNote">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>便利贴：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTaskManager">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>任务管理器：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelColorPicker">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>颜色拾取：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHotkeyManager">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>热键管理：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelReminder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>定时提醒：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHardware">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>查看硬件信息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStock">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>自选股：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMainTool">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>实用工具：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTranslator">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelHardwareTranslator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>划词翻译：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSearch">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelSearch">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索工具：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelScreenShotTip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelStickyNoteTip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTaskManagerTip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelMainToolTip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelColorPickerTip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTranslatorTip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelHotkeyManagerTip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelStockTip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelHardwareTip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelReminderTip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelSearchTip">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShortcutSetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShortcutSetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
